modnet/models.py

```diff
@@ -1,30 +1,44 @@
 import pickle
 from typing import List, Tuple, Dict, Optional, Callable, Any
 
+from pathlib import Path
+
 import pandas as pd
 import numpy as np
 from sklearn.preprocessing import StandardScaler, MinMaxScaler
-import tensorflow.keras as keras
+from sklearn.utils import resample
+from sklearn.model_selection import train_test_split
+
+import tensorflow as tf
+tf.config.threading.set_intra_op_parallelism_threads(1)
+tf.config.threading.set_inter_op_parallelism_threads(1)
+import os
+os.environ["CUDA_VISIBLE_DEVICES"] = "-1"
+os.environ["OMP_NUM_THREADS"] = "1"
 
+import tensorflow_probability as tfp
```

Comment:
```suggestion
try:
    import tensorflow_probability as tfp
except ImportError:
    pass
```

I would suggest that we make the tf probability stuff optional, probably in a different class

Reply:
We could import it inside BayesianModnetModel